packages/std/src/transactions.rs

```diff
@@ -102,10 +104,13 @@ impl RepLog {
     }
 
     /// applies the stored list of `Op`s to the provided `Storage`
-    pub fn commit<S: Storage>(self, storage: &mut S) {
+    pub fn commit<S: Storage>(self, storage: &mut S) -> Result<()> {
         for op in self.ops_log {
-            op.apply(storage);
+            if let Err(err) = op.apply(storage) {
```

Comment:
Replace this if statement with: 
`op.apply(storage)?`


Or `let _ = op.apply(storage)?` if needed

Reply:
Done in 6fd1dbe481e97a77c8c785cb95179a46f01557ef